_generated/KubeloginInstallerV0_Node20/utils.ts

```diff
@@ -47,13 +47,23 @@ export function isLatestVersion(version: string): boolean {
   return v === 'latest' || v === '*' || v === '';
 }
 
+function addAuthorizationHeaderIfEnabled(request: webClient.WebRequest): void {
+  if (taskLib.getBoolFeatureFlag('USE_AUTHORIZATION_FOR_API_CALL')) {
+    const token = getGithubEndPointToken();
+    if (token) {
```

Comment:
It will be great to see some warning in case there is empty token

Reply:
Added the warnng